Source/WebCore/platform/graphics/FontPlatformData.cpp

```diff
@@ -114,4 +115,68 @@ Vector<FontPlatformData::FontVariationAxis> FontPlatformData::variationAxes(Shou
 }
 #endif
 
+#if USE(CORE_TEXT)
+bool FontPlatformData::canMakeBaseFontDescriptor(const Vector<uint8_t>& fontFaceData, String& itemInCollection)
+{
+    Vector<uint8_t> copyOfFontFaceData = fontFaceData;
+    auto localFontFaceData = SharedBuffer::create(WTFMove(copyOfFontFaceData));
+    auto fontCustomPlatformData = createFontCustomPlatformData(localFontFaceData, itemInCollection);
```

Comment:
This is a lot of unnecessary copying and duplicate work just for a verifier.  Could we instead just try making the object and validate that it succeeded?